sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -890,7 +890,7 @@ primaryExpression
     | OVERLAY '(' input=valueExpression PLACING replace=valueExpression
       FROM position=valueExpression (FOR length=valueExpression)? ')'                          #overlay
     | PERCENTILE_CONT '(' percentage=valueExpression ')'
-      WITHIN GROUP '(' ORDER BY sortItem ')'                                                   #percentile
+      WITHIN GROUP '(' ORDER BY sortItem ')' ( OVER windowSpec)?                               #percentile
```

Comment:
I'd like to understand the rationale more. This function has its own syntax to specify ORDER BY, so I get that it should not co-exists with the ORDER BY in the window frame. But doesn't `percentile` have the same problem when being used as a window function?

Reply:
And why can't users specify the window frame boundaries?

Reply:
According to the document of these database, Almost all databases tell me the window specification of `PERCENTILE_CONT` only have partition by.
I can't find any other words about root cause or rationale.